migration/1582875306439-note-reaction-length.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class noteReactionLength1582875306439 implements MigrationInterface {
+    name = 'noteReactionLength1582875306439'
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(`ALTER TABLE "note_reaction" ALTER COLUMN "reaction" TYPE character varying(130)`, undefined);
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(`ALTER TABLE "note_reaction" ALTER COLUMN "reaction" TYPE character varying(128)`, undefined);
```

Comment:
マイグレーション的にマイグレーションする前はlength=128ではなく32なはずなのでそっちにしたほうが良い気がする (まあそもそもdownするかと言われると…という話はあるけど)

```suggestion
        await queryRunner.query(`ALTER TABLE "note_reaction" ALTER COLUMN "reaction" TYPE character varying(32)`, undefined);
```

Reply:
この前のマイグレスクリプト (Init) だと128みたいです
https://github.com/syuilo/misskey/blob/d3fd0f810a73df5a4a354413b8a8914eb1949ff4/migration/1000000000000-Init.ts#L61

32でデプロイされている環境は、上のInitスクリプトが作られる前に作られた環境だと思われます。